strongbox-web-core/src/main/java/ru/sbespalov/test/Game.java

```diff
@@ -0,0 +1,51 @@
+package ru.sbespalov.test;
+
+import java.util.Collections;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Game
```

Comment:
@sbespalov it looks like this is not supposed to be here or am I missing something? :)

Reply:
removed :)